rpm/build-config

```diff
@@ -1,46 +1,643 @@
-CONFIG_CTRL_IFACE=y
-CONFIG_CTRL_IFACE_DBUS=y
```

Comment:
This is also missing. I wonder do the `CONFIG_CTRL_IFACE_DBUS_{NEW,INTRO}` enable this then.

Reply:
Yes it does:
https://github.com/sailfishos-mirror/hostap/commit/8fc2fb56e456b4dfea4ea7c4afc34c20309560cd

Reply:
Right. "Nothing should have been using this since 2010" :D.